IWYU: stdint.h in pdfium_mem_buffer_file_write.h for uint8_t

Bug: 879900
Change-Id: I9c15d1c280a23c53d31f2d72c9d0d1db79eab886
Reviewed-on: https://chromium-review.googlesource.com/1204410
Reviewed-by: Lei Zhang <thestig@chromium.org>
Commit-Queue: Henrique Nakashima <hnakashima@chromium.org>
Cr-Commit-Position: refs/heads/master@{#588547}
diff --git a/pdf/pdfium/pdfium_mem_buffer_file_write.h b/pdf/pdfium/pdfium_mem_buffer_file_write.h
index 03c54bb..82e82d2 100644
--- a/pdf/pdfium/pdfium_mem_buffer_file_write.h
+++ b/pdf/pdfium/pdfium_mem_buffer_file_write.h
@@ -6,6 +6,7 @@
 #define PDF_PDFIUM_PDFIUM_MEM_BUFFER_FILE_WRITE_H_
 
 #include <stddef.h>
+#include <stdint.h>
 
 #include <vector>
